src/bridging/providers/bungee/BungeeBridgeProvider.ts

```diff
@@ -191,6 +194,8 @@ export class BungeeBridgeProvider implements BridgeProvider<BungeeQuoteResult> {
       chainId,
       signer,
       defaultGasLimit,
+      deadline: getOrderDeadlineFromNow(BRIDGE_HOOK_VALIDITY),
```

Comment:
This is an absolute deadline, right?
Also, it's based on the `DEFAULT_QUOTE_VALIDITY`.

What happens with orders with longer expiration dates?
Would it be possible to use here the real order deadline as reference and add to that instead?

Reply:
As we discussed, I got rid of `BRIDGE_HOOK_VALIDITY_THRESHOLD `.
Now deadline = order.deadline